From: Balint Reczey <balint.reczey@canonical.com>
Date: Wed, 28 Nov 2018 20:39:26 +0100
Subject: Tests: Skip test for empty cypher string

OpenSSL does not throw error now, OTOH this is the expected behavior.
But this would make running the test suite failing.

Forwarded: https://github.com/openssl/openssl/issues/7725
---
 src/twisted/test/test_sslverify.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/twisted/test/test_sslverify.py b/src/twisted/test/test_sslverify.py
index faae0ac..e9725ab 100644
--- a/src/twisted/test/test_sslverify.py
+++ b/src/twisted/test/test_sslverify.py
@@ -2789,7 +2789,7 @@ class ExpandCipherStringTests(TestCase):
     if skipSSL:
         skip = skipSSL
 
-    def test_doesNotStumbleOverEmptyList(self):
+    def _test_doesNotStumbleOverEmptyList(self):
         """
         If the expanded cipher list is empty, an empty L{list} is returned.
         """
